pyfstat/make_sfts.py

```diff
@@ -214,6 +225,22 @@ def basic_setup(self):
         self.config_file_name = os.path.join(self.outdir, self.label + ".cff")
         self.theta = np.array([self.phi, self.F0, self.F1, self.F2])
 
+        required_signal_params = [
+            # leaving out "F1","F2","psi","phi","tref" as they have defaults
```

Comment:
I understand why `F1, F2` have zero as default (They are not required for every kind of CW signal);`tref` comes from `tstart` unless explicitly given; `h0` and `cosi` set the "overall amplitude" of the CW (if `h0` has to be set, makes sense to set `cosi` as well).

Any criterion on why `psi, phi` are also set to zero other that "if you don't set them then you (most likely) don't care about them since Tobs will be long enough to swap out any bias"?

Reply:
Scientific answer: because, unlike cosi, fixing psi and phi to 0 won't bias injection campaigns.

Pragmatic answer, I'd say this is the stronger one: because MFDv5 itself will throw an error if h0=0 but no explicit cosi given, while it also silently defaults to psi=phi0=0.

Reply:
Thanks